docs/admin/templates/extending-templates/parameters.md

```diff
@@ -287,7 +287,10 @@ data "coder_parameter" "project_id" {
 }
 ```
 
-## Create Autofill
+## Autofill Parameters
+
+> This feature requires the `auto-fill-parameters`
+> [experiment](https://coder.com/docs/contributing/feature-stages).
```

Comment:
This link should point to its new home from `restructure-new` branch. Looks like we do not have this yet.